tests/p2p/test_split_brain.py

```diff
@@ -13,21 +15,20 @@
 class BaseHathorSyncMethodsTestCase(unittest.TestCase):
     __test__ = False
 
-    def setUp(self):
+    def setUp(self) -> None:
         super().setUp()
 
         first_timestamp = self._settings.GENESIS_BLOCK_TIMESTAMP
         self.clock.advance(first_timestamp + self.rng.randint(3600, 120*24*3600))
 
         self.network = 'testnet'
 
-    def create_peer(self, network, unlock_wallet=True):
+    def create_peer(self, network: str, unlock_wallet: bool = True) -> HathorManager:  # type: ignore[override]
         wallet = HDWallet(gap_limit=2)
         wallet._manually_initialize()
 
         manager = super().create_peer(network, wallet=wallet)
         manager.daa.TEST_MODE = TestMode.TEST_ALL_WEIGHT
-        manager.avg_time_between_blocks = 64
```

Comment:
Why are you changing this?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have a `avg_time_between_blocks` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
Weird. I thought these tests were changing this parameter to get more blocks.

Reply:
I believe that this test was designed to cover a case with a different average time between blocks. We might add an `avg_time_between_blocks: iint` to the `create_peer()` method, pass the modified value here, and refactor there to make it work accordingly (or add a TODO to get this done later).

Reply:
I agree that this shouldn't be just removed. I'm leaning more towards commenting this line and adding a FIXME just to keep this PR purely as a refactor. Since the line currently has no effect, changing it is at least a `fix`/`test` change, that may or may not need more changes to make the test pass, but at the very least it will change the behavior of the test (for the better since it's the original intent that was once working before).

Reply:
Ok, I commented all the lines and added a `FIXME` in https://github.com/HathorNetwork/hathor-core/pull/973/commits/20a973dcba804e43868c583e811af6c878a21f22

Reply:
Thanks!